langgraph_bot/tools/tools.py

```
@@ -0,0 +1,31 @@
+from typing import Optional
+from models.generativemodel import groqmodel
+from utils.prompts import TITLE_PROMPT
+from langchain.tools import  tool,ToolRuntime
+from langchain.messages import ToolMessage
+from langgraph.types import Command
+postdata= "demo"
+posttitle ="demo"
+
+
+@tool
+def title_tool(posttitle:str,postdata:str,runtime:ToolRuntime):
```

Ruff (0.14.8)

Unused function argument: `runtime`

(ARG001)